inference-engine/src/mkldnn_plugin/utils/node_dumper.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#ifdef CPU_DEBUG_CAPS
+#pragma once
+
+#include "mkldnn_node.h"
+
+#include <array>
+#include <string>
+
+namespace MKLDNNPlugin {
+
+/**
+ * Blobs are not dumped by default
+ * Blobs are dumped if node matches all specified env filters
+ *
+ * To dump blobs from all the nodes use the following filter:
+ *
+ * OV_CPU_BLOB_DUMP_NODE_NAME=.+
+ */
+class NodeDumper {
+public:
+    NodeDumper(int _count) : count(_count) {
+        setup();
+    }
+
+    void setup();
+
```

Comment:
Seem that this is an internal utility function, m.b. it would be better to move it to the private section.

Reply:
Will be corrected